examples/page-visit-counter-tailwindcss/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="/tailwind.css" type="text/css" rel="stylesheet" />
+    <title>Http Example App</title>
+
+    <link data-trunk rel="scss" href="index.scss" />
```

Comment:
```suggestion
    <link data-trunk rel="scss" href="index.scss" />
```
We can probably delete this since we're using `tailwind.css` already.